# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#current position and orientation
geometry_msgs/Pose2D res_pos
float64 dis
